docs/asyncapi.yaml

```diff
@@ -0,0 +1,92 @@
+asyncapi: 3.0.0
+info:
+  title: FCP-FD-COMMS
+  version: 1.0.0
+  description: Receive a communication message and route to appropriate channels
+
+channels:
+  sourceSystem-fcp-fd-comms:
+    address: sourceSystem-fcp-fd-comms
+    description: |
+      Accepts POST requests for communication notifications.
+    messages:
+      subscribe.message.0:
+        $ref: '#/components/messages/notifyMessage'
+
+operations:
+  SourceSystem/fcp-fd-comms.subscribe:
+    action: receive
+    channel:
+      $ref: '#/channels/sourceSystem-fcp-fd-comms'
+
+components:
+  messages:
+    notifyMessage:
+      title: gov.notify message
+      contentType: application/json
+      payload:
+        $ref: '#/components/schemas/NotificationPayload'
+
+  schemas:
+    NotificationPayload:
+      type: object
+      properties:
+        CRN:
+          type: number
+          description: Customer Reference Number
+          minimum: 105000000
+          maximum: 999999999
+          example: 123456789
+        SBI:
+          type: number
+          description: Single Business Identifier
+          minimum: 105000000
+          maximum: 999999999
+          example: 123456789
+        sourceSystem:
```

Comment:
We need email address too

Reply:
It's a good point.  I don't have a definitive answer, but here's my conflicting views.

We only support email, so if we moved it a level up, then validation of inbound messages is easier as we don't need to delve into personalisation which is template specific.

It's not totally unexpected that we may support other communication methods through this contract and therefore, having email as a required top level field would introduce a breaking change later

We've already got `commsType`.  Rather than a fixed `email` field, we could have `commsAddress` (or something) which if email we'd want an email address there, if sms, we'd want a mobile number etc.

Further consideration, does this handle a range of email addresses being supplied?  In which case, it really is personalisation (does Notify even support that?)

Reply:
Really good suggestion around commsAddress for future-proofing. Taking this onboard